workers/unity/Packages/com.improbable.gdk.movement/Behaviours/CharacterControllerMotor.cs

```diff
@@ -109,17 +117,22 @@ public virtual void MoveWithExtensions(Vector3 toMove)
             }
         }
 
-        protected bool CheckOverrideInAir()
+        protected void CheckGrounded()
+        {
+            IsGrounded = Physics.OverlapSphereNonAlloc(transform.position, groundedRadius, groundedOverlapSphereArray,
+                groundLayerMask) > 0;
+        }
+
+        protected void CheckOverrideInAir()
```

Comment:
Confused by the name of this method. What's the meaning of "Override" in this context? Was this an already existing method name?

Reply:
This was, yeah. So the extensions can force the motor to be considered as not grounded, despite the ground checker detecting it and passing. I can rename this if something would be preferable?
(CheckExtensionsForOverrides?)

Reply:
Hmm, yeah, that makes a bit more sense I guess.